components/MVP/Header/Header.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import classnames from 'classnames'
+import {
+  Header as USWDSHeader,
+  Title,
+  NavMenuButton,
+  PrimaryNav,
+  Menu,
+  NavDropDownButton,
+  NavList,
+} from '@trussworks/react-uswds'
+
+import styles from './Header.module.scss'
+
+const Header = () => {
+  const router = useRouter()
+  const [mobileNavOpen, setMobileNavOpen] = useState(false)
+  const [servicePortalNavOpen, setServicePortalNavOpen] = useState(false)
+
+  const isHomePage = router.pathname === '/'
+
+  const handleToggleMobileNav = () => {
+    setMobileNavOpen(!mobileNavOpen)
+  }
+
+  const handleToggleServicePortalMenu = () => {
+    setServicePortalNavOpen(!servicePortalNavOpen)
+  }
+
+  const navItems = [
+    { path: '/', label: 'Home' },
+    { path: '/news', label: 'News', className: styles.newsLink },
+    {
+      path: '/training-and-education',
+      label: 'Training and education',
+      className: styles.trainingLink,
+    },
+    { path: '/about-us', label: 'About us', className: styles.aboutLink },
+  ].map((i) => (
+    <Link href={i.path} key={i.path}>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <a
+        className={classnames('usa-nav__link', i.className, {
+          'usa-current': i.path === router.pathname,
+        })}>
+        <span>{i.label}</span>
+      </a>
+    </Link>
+  ))
+
+  const servicePortalItems = [
+    { path: 'https://www.hrcapps.army.mil/portal/', label: 'Army' },
+    { path: 'https://my.navy.mil/', label: 'Navy' },
+    { path: 'https://www.my.af.mil/', label: 'Air Force' },
+    { path: 'https://mol.tfs.usmc.mil/', label: 'Marines' },
+    { path: 'https://cgportal2.uscg.mil/', label: 'Coast Guard' },
+  ].map((i) => (
+    <Link href={i.path} key={i.path}>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <a>{i.label}</a>
+    </Link>
+  ))
+
+  const homeNavItems = [
+    { path: '/#manage-your-life', label: 'Manage your life' },
+    { path: '/#work-tools', label: 'Work tools' },
+  ].map((i) => (
+    <Link href={i.path} key={i.path}>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <a className="usa-nav__link">{i.label}</a>
+    </Link>
+  ))
+
+  const servicePortalMenu = (
```

Comment:
This is existing behavior in the deployed mvp, but the dropdown isn't trapping focus. Would we like to do that moving forward when building any dropdowns? Curious what the best practice is.

Reply:
we talked about this, the same issue occurs in USWDS dropdowns. For future development we may look into using a focus trap for elements like this.